src/__tests__/DisplayArticles.spec.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { shallow } from "enzyme";
+import axios from "axios";
+
+import DisplayArticles from "../components/DisplayArticles";
+
+describe("<DisplayArticles />", () => {
+  it("Should fetch articles from API using Axios", () => {
+    const axiosSpy = jest.spyOn(axios, "get");
+    shallow(<DisplayArticles />);
+    expect(axiosSpy).toBeCalled();
```

Comment:
Yes, we've been down this route before. Is there anything else you can test for in this component?